evansmusomi/burger-builder
# Adds Custom Form Validation and Feedback

*What this does*
- Adds custom validation on contact data form
- Adds visual validation feedback on contact data form

*How to verify*
- User can see fields with invalid data on the contact data form
- User can only submit order when all fields on the form are valid
